### PR TITLE
add new changes to multiple files

### PR DESCRIPTION
**Pull request by IlaganJireh**
Hello, I have changes in my files that needs reviewing, thank you!

### CODE REVIEW

**Comment by nabayra-james** [comment]
Hello, Andrei! I'm currently reviewing your files. I will keep you updated here if there's any requested changes. Thank you for your patience!

File: ilagan_andreijireh/index.html [hunk 1]
```diff
@@ -183,6 +183,16 @@ <h3>Add a comment here</h3>
         <h3>Comments</h3>
       </div>
 
+      <div class="container">
+        <h1>Country Information</h1>
+        <div class="search-box">
+            <input type="text" id="country-input" placeholder="Enter country name">
```
Comment (nabayra-james) [inline ilagan_andreijireh/index.html]: ### IDs should be written in snake case. All lower case separated by underscore. 

References: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

This also applies in lines 190, 192, and 193 in your HTML file.

File: ilagan_andreijireh/script.js [hunk 2]
```diff
@@ -91,4 +91,37 @@ function loadComments() {
     comments = JSON.parse(storedComments);
     displayComments();
   }
-}
\ No newline at end of file
+}
+
+document.getElementById('search-button').addEventListener('click', function() {
```
Comment (nabayra-james) [inline ilagan_andreijireh/script.js]: ### IDs should be written in snake case. All lower case separated by underscore.

Of course, when you commit changes to your html, you should edit your js file too for it to get the appropriate id from your HTML file.

This also applies in lines 97, 101, and 117.

File: ilagan_andreijireh/script.js [hunk 3]
```diff
@@ -91,4 +91,37 @@ function loadComments() {
...
+  var country = document.getElementById('country-input').value;
+  fetch('https://restcountries.com/v3.1/name/' + country)
```
Comment (nabayra-james) [inline ilagan_andreijireh/script.js]: ### Use guard clause here when possible to catch early errors

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d183194fb_0_0

You can just do this instead:
.then((response) => {
      if (!response.ok) {
        throw new Error('Country not found');
      }
      return response.json();
    })

File: ilagan_andreijireh/script.js [hunk 4]
```diff
@@ -91,4 +91,37 @@ function loadComments() {
     comments = JSON.parse(storedComments);
     displayComments();
   }
-}
\ No newline at end of file
+}
+
+document.getElementById('search-button').addEventListener('click', function() {
+  var country = document.getElementById('country-input').value;
```
Comment (nabayra-james) [inline ilagan_andreijireh/script.js]: ### Change the var keyword to let

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d183194fb_0_0

**Review by nabayra-james — CHANGES_REQUESTED** [review]

**Comment by IlaganJireh** [comment]
Hello, I've made a new pull request for the changes. Thank you.

**Review by nabayra-james — APPROVED** [review]

**Comment by IlaganJireh** [comment]
Hello, huge thanks for your patience, thank you for approving my request!

**Pull request merged**